.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 .idea
 .build
+publis/js/_config.js
```

Comment:
because postinstall will generate `_config.js` via the `put-packages-in-requirejs-config` task and we don't want it in version control